scipy/stats/morestats.py

```diff
@@ -3051,13 +3051,13 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
                 # r_plus is the center of the distribution.
                 prob = 1.0
             else:
-                p_less = np.sum(cnt[:r_plus + 1]) / 2**count
-                p_greater = np.sum(cnt[r_plus:]) / 2**count
+                p_less = np.sum(cnt[:r_plus + 1])
```

Comment:
Maybe the variable name `cnt` should change now that it's probabilities instead of frequencies?